frontend/portals/NavbarNavigatorCartButtonBefore/index.jsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import { ITEM_PATTERN } from '@shopgate/pwa-common-commerce/product/constants';
-import { withPageState } from '@shopgate/pwa-extension-kit/connectors';
-import isIOSTheme from '@shopgate/pwa-extension-kit/env/helpers/isIOSTheme';
+import { withPageState } from '@shopgate-ps/pwa-extension-kit/connectors';
+import isIOSTheme from '@shopgate-ps/pwa-extension-kit/env/helpers/isIOSTheme';
 import ShareButton from '../../components/ShareButton';
 
-const NavbarNavigatorCartButtonBefore =  ({ pattern, ...otherProps }) => {
+const NavbarNavigatorCartButtonBefore = ({ pattern, ...otherProps }) => {
```

Comment:
the linter is complaining because there is no prop validation for pattern and missing jsDoc

Reply:
Sorry, don't think i opened this file since i only worked on unit tests. corrected linter issues